udata_front/theme/gouvfr/assets/js/components/MultiSelect/MultiSelect.vue

```diff
@@ -651,26 +651,22 @@ export default defineComponent({
     };
 
     function diff(value1, value2) {
-      if (typeof value1 === 'string' && typeof value2 === 'string') {
-        return value1 === value2 ? "" : value1;
-      } else if (typeof value1 === 'object' && typeof value2 === 'object') {
+      if (value1.isArray && value2.isArray) {
```

Comment:
Array.isArray() is a function taking a param see https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray